tests/test_run_service.py

```
@@ -533,6 +572,7 @@ def test_list_runs_paginated_with_flow_key_filter(self, tmp_path, monkeypatch):
         # Mock storage functions to use only our test runs
         run_ids = ["run-signal", "run-build"]
         monkeypatch.setattr(storage, "list_runs", lambda runs_dir=None: run_ids)
```

Ruff (0.15.4)

[warning] Unused lambda argument: `runs_dir`

(ARG005)